scripts/grafana-server/custom.ini

```diff
@@ -1,5 +1,11 @@
+app_mode = test
```

Comment:
editing this file seems wrong.... there is a file `conf/custom.ini` that will update your environment but is not checked into git

Reply:
Is there some other place where feature toggles should be updated for the testing environment?

Reply:
my bad -- i did not realize this is the e2e scripts folder